src/cli/cluster/cluster_manager.js

```diff
@@ -163,6 +163,7 @@ export default class ClusterManager {
     const { fromRoot } = require('../../legacy/utils');
 
     const watchPaths = [
+      fromRoot('src/core'),
```

Comment:
Should this be `src/core/server` to prevent server restarts for client-side changes?

Reply:
I tested this and it doesn't restart the server for client-side changes (same for `src/legacy/ui`a few lines down).

I want the entire `core` directory to be watched because there are a couple directories for shared code as well (though this may change in the near future).

Reply:
We must be ignoring public directories somewhere?